tools/build-release-branch.sh

```diff
@@ -52,6 +52,7 @@ function create_release_gitignore {
 
     # Add custom stuff to .gitignore release
     echo "/_inc/client" >> .gitignore
+    echo "/extensions/**/*.{js,jsx,json,gif,png,css,jpg,jpeg,sass,scss,svg}" >> .gitignore
```

Comment:
Maybe add a note that there's a line in `.svnignore` that needs to be synced with this (or vice versa)?

Reply:
eb96cbe